api/reference/automation_requests.adoc

```diff
@@ -41,6 +42,68 @@ As an example:
 }
 ----
 
+[[editing-requests]]
+=== Editing Requests
+
+Updating requests can be performed by posting an _edit_ action as follows:
+
+[source,data]
+----
+POST /api/automation_requests/11
+----
+
+[source,json]
+----
+{
+  "action" : "edit",
+  "resource" : {
+    "options" : {
+      "attrs" : {
+        "var1" : "updated value1",
+        "var2" : "updated value2",
+        "minimum_memory" : 4096
+      }
+    }
+  }
+}
+----
+
+Response:
+
+[source,json]
+----
+{
+  "href": "http://localhost:3000/api/automation_requests/11",
```

Comment:
In other parts of the documentation an ID is referenced with `:id`, should that stay consistent?

Reply:
For snippets that have Request/Response examples I tend to have actual numbers instead of the generic :id for usage.